src/web/web_panel.cpp

```
@@ -88,19 +88,19 @@ WebPanel::WebPanel() {
   _server->SetCheckConnectionCallback(
-      [](ws28::Client *client, ws28::HTTPRequest &) { return true; });
+      [](ws28::Client* client, ws28::HTTPRequest&) { return true; });
 
   _server->SetClientConnectedCallback(
-      [](ws28::Client *client, ws28::HTTPRequest &) {
+      [](ws28::Client* client, ws28::HTTPRequest&) {
         spdlog::get("web")->debug("Client connected from {}", client->GetIP());
 
-        auto *data = (ClientData *)malloc(sizeof(ClientData));
+        auto* data = (ClientData*)malloc(sizeof(ClientData));
         data->authed = false;
```

Cppcheck (2.20.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)